src/test/java/scenarios/nativeHomeworkTest.java

```diff
@@ -0,0 +1,47 @@
+package scenarios;
+
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import setup.BaseTest;
+
+public class nativeHomeworkTest extends BaseTest {
```

Comment:
First letter in a class name must be in upper case NativeHomeworkTest

Reply:
Fixed